maven/bnd-export-plugin/README.md

```diff
@@ -0,0 +1,28 @@
+# bnd-export-maven-plugin
+
+A plugin to optionally resolve and export bndrun files.
+
+Point the plugin to a bndrun file in the same project. It will optionally resolve the project and export
+a runnable jar into the targetDir. If not specified targetDir defaults to the project build directory.
+
+```
+            <plugin>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-export-maven-plugin</artifactId>
+                <version>3.2.0-SNAPSHOT</version>
+                <configuration>
+                    <bndrun>
```

Comment:
Is this correct since multiple bndrun files can be specified? I thought it would be <bndruns><bndrun>file</bndrun></bndruns>.